src/tests/Common/Platform/platformdefines.h

```diff
@@ -72,9 +73,11 @@ typedef WCHAR *LPWSTR, *PWSTR;
 typedef const WCHAR *LPCWSTR, *PCWSTR;
 
 typedef int HRESULT;
-#define LONGLONG long long
-#define ULONGLONG unsigned LONGLONG
-typedef unsigned int ULONG, *PULONG;
+
+typedef unsigned long long ULONGLONG;
```

Comment:
Why unsigned long long here and not uint64_t?

Reply:
That is a good question. I didn't even think about it. I will say this is in a higher level header file so it might impact others. I will give it a shot though, it is a good suggestion.